tests/actions/orderEntryActions.test.js

```diff
@@ -61,14 +61,42 @@ describe ('Order entry configuration actions', () => {
             });
         });
 
-        it ('should dispatch `FETCH_ORDER_CONFIG_FAILURE` after an error', () => {
+        it ('should dispatch `FETCH_ORDER_CONFIG_FAILURE` if status is not 200', () => {
             moxios.wait(() => {
                 const request = moxios.requests.mostRecent();
                 request.respondWith({
+                    status: 204,
+                    response: {
+                        statusText: 'OK'
+                    },
+                });
+            });
+
+            const expectedActions = [
+                FETCH_ORDER_CONFIG_LOADING,
+                FETCH_ORDER_CONFIG_LOADING,
+                FETCH_ORDER_CONFIG_FAILURE
+            ];
+
+            const store = mockStore({ configurations: []});
+
+            return store.dispatch(getOrderEntryConfigurations()).then(() => {
+                const dispatchedActions = store.getActions();
+                const dispatchedActionTypes = dispatchedActions.map(action => action.type);
+                expect(dispatchedActionTypes).toEqual(expectedActions);
+            });
+        });
+
+
+        it ('should dispatch `FETCH_ORDER_CONFIG_FAILURE` after an error', () => {
+            moxios.wait(() => {
+                const request = moxios.requests.mostRecent();
+                request.reject({
                     status: 401,
```

Comment:
Here you have the same error dispatch but a status of `401`

Reply:
401 is the status code for unauthorized and axios catches this as an error explaining the error dispatch.

Reply:
If this error occurs due to unauthorized access, could you indicate this in the `it` description?
This would help make it clearer.

Reply:
noted, thank you.